website/docs/r/control_mute.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "turbot"
+title: "turbot"
+template: Documentation
+page_title: "Turbot: turbot_control_mute"
+nav:
+  title: turbot_control_mute
+---
+
+# turbot\_control\_mute
+
+`Turbot Control Mute` allows to mute a control on a given resource.
+
+## Example Usage
+
+**Mute a Control**
+
```

Comment:
Could we please add more examples for common use-cases?

Reply:
Could we add another example of muting a control only in error state?
Example 1: Suppress alarm and error for an `AWS > EC2 > Instance > Approved` control.
Example 2: Suppress error for an `AWS > S3 > Bucket > Tags` control.